lib/waffle_ecto/schema.ex

```diff
@@ -104,6 +104,18 @@ defmodule Waffle.Ecto.Schema do
       when is_binary(filename) and is_binary(binary) ->
         [{field, {upload, scope}} | fields]
 
+      {field, data = "data:image/" <> _}, fields ->
+        encoded_image = String.replace(data, ~r/^data\:image\/.*;base64,/, "")
+
+        case Base.decode64(encoded_image) do
```

Comment:
I looking for something similar and maybe it's worth to have special key like Arc
something like that `%{filename: filename, base64: base64}`

stolen from https://github.com/stavro/arc/pull/108/files